test/unit/types/array_type_test.rb

```diff
@@ -1,8 +1,8 @@
 require 'test_helper'
 
 class Superstore::Types::ArrayTypeTest < Superstore::Types::TestCase
-  test 'typecast' do
```

Comment:
Do you think it makes sense to alias the `typecast` and `cast_value` along with `encode` & `serialize`?  It would be nice to avoid breaking the api.

Reply:
I am pretty sure that our own app is the only user of that API, and this PR is a major version change for superstore.

Reply:
I just mean that if you alias them, then you can upgrade the gem without other changes and test to see that things still behave.  Then you can change our apps to use the new code, and then remove the aliases.  It makes the upgrade more atomic.  But if you wanna do it all in one big go, be my guest! 😄